### PR TITLE
Update kits and items to include new stuff

### PR DESCRIPTION
**Pull request by jedediah**

### CODE REVIEW

File: src/modules/items.haml [hunk 1]
```diff
@@ -367,36 +406,78 @@
                     <br/>
                     ### [Enchantments](id:enchantments)
 
-                    Items and armor can have custom enchantments by using the `enchantment` attribute. Enchantments are specified by their [bukkit name](/reference/enchantments) and the enchantment level separated by a colon `:`. Multiple enchantments are separated by a semicolon `;`.
+                    Any enchantment can be applied to any item. The enchantment type can be specified by
+                    [Minecraft name](http://minecraft.gamepedia.com/Data_values#Enchantment_IDs) or
+                    [Bukkit name](/reference/enchantments).
```
Comment (McSpider) [inline src/modules/items.haml]: IMO, this sentence doesn't sound grammatical correct.
Maybe use "Minecraft enchantment names" & "Bukkit enchantment names" to match the sentence for potion effects, or, "can be specified by their".

File: src/modules/kits.haml [hunk 2]
```diff
@@ -10,10 +10,28 @@
...
+                    Kits are groups of items and effects that can be given to players in various ways. There are many types of kits, and they can be
+                    packaged together in any combination.
 
-                    Note that by default a kit will not empty a inventory but rather overwrite specific slots. If you want to guarantee that a slot is empty use air with an amount of 0 as the material for that slot. A player's whole inventory or part of it can also be cleared by using one of the clear elements.
+                    Kits can be given an `id` attribute, which is used to refer to them in other places.
+                    Kits can contain other kits directly, and can also inherit items from other kits through the `parents=""` attribute.
 
+                    Some types of kits are *removable*, meaning that the kit can be automatically taken from the player as well as given. When used in certain contexts,
+                    these kits will be automatically removed from the player at an appropriate time. This will be noted in the documentation where applicable.
```
Comment (McSpider) [inline src/modules/kits.haml]: The contexts in which a kit is removed currently don't appear to be documented?

File: src/modules/potions.haml [hunk 3]
```diff
@@ -21,54 +24,43 @@
                         %thead
                             %tr
                                 %th Element
+                                %th Attribute
                                 %th Description
-                                %th Value
+                                %th Default
```
Comment (McSpider) [inline src/modules/potions.haml]: I still think doing the table this way looks ugly and restricts the length of attribute descriptions (not really a problem in this case).

**Comment by McSpider** [comment]
Other what what I've noted everything else looks good :) (Much better than what I would have written)

File: src/modules/items.haml (hunk 1 above)
Comment (jedediah) [inline src/modules/items.haml]: improved

File: src/modules/kits.haml (hunk 2 above)
Comment (jedediah) [inline src/modules/kits.haml]: added

File: src/modules/potions.haml (hunk 3 above)
Comment (jedediah) [inline src/modules/potions.haml]: fixed

**Pull request merged**